docs/content/configuration/creating_volumesnapshotclass_topology_aware.md

```diff
@@ -41,4 +41,5 @@ After the YAML file is created, apply it by using the `kubectl apply -f` command
 kubectl apply -f <filename>.yaml
 ```
 
- The `volumesnapshotclass.snapshot.storage.k8s.io/<volumesnapshotclass-name> created` message is emitted.
\ No newline at end of file
+ The `volumesnapshotclass.snapshot.storage.k8s.io/<volumesnapshotclass-name> created` message is emitted.
+ 
```

Comment:
do we want this extra new line?

Reply:
I think this means there's one extra line in addition to what github likes. please try to remove 1

Reply:
done

Reply:
thanks. ok, now there's no apparent additional line, but there's still a change (same as in the [limitations file](https://github.com/IBM/ibm-block-csi-driver/pull/458#discussion_r812993257)).
could we try the same additional single line deletion here?

Reply:
deleted